src/test/ui/issues/issue-50865-private-impl-trait/auxiliary/lib.rs

```diff
@@ -1,3 +1,6 @@
+// compile-flags: -Z mir-opt-level=2
```

Comment:
can you use revisions here, so that we keep testing it once without `mir-opt-level=2`?

Reply:
Good point, will do.